Specs/Scene/PointAppearanceSpec.js

```diff
@@ -0,0 +1,139 @@
+/*global defineSuite*/
+defineSuite([
```

Comment:
Yeah they are usually split up this way. The geometry specs are simpler just checking default values and helper functions, and the appearance spec does the rendering.